backend/penndata/views.py

```diff
@@ -288,7 +288,7 @@ def get(self, request):
 
 class FitnessUsage(APIView):
     def safe_add(self, a, b):
-        return a + b if a and b else (a if a else b)
+        return None if a is None and b is None else (a or 0) + (b or 0)
```

Comment:
this is the only real change. everything else was just me refactoring. 

i'm pretty sure the problem was when a or b was 0 and the other None. then safe_add didn't behave as expected and could return None (depending on order of a, b). 

and then you get None when you call this twice so safe_add(None, 0) = 0, but then safe_add(0, None) = None.